snap/info.go

```diff
@@ -311,6 +312,9 @@ type Info struct {
 	// List of system users (usernames) this snap may use. The group of the same
 	// name must also exist.
 	SystemUsernames map[string]*SystemUsernameInfo
+
+	// OriginalLinks is a map links keys to link lists
```

Comment:
minor: I find this easier to read (since link is also a verb):
```suggestion
	// OriginalLinks is a map linking keys to lists of links
```